packages/desktop-client/src/components/manager/subscribe/ChangePassword.tsx

```diff
@@ -30,31 +33,33 @@ export function ChangePassword() {
     }
   }
 
-  async function onSetPassword(password) {
+  async function onSetPassword(password, t: (key: string) => string) {
```

Comment:
Same comment here, can we use `getI18n` here to get the `t` from there instead of passing the `t` as parameter?